recipe/meta.yaml

```diff
@@ -1,13 +1,13 @@
 {% set name = "HepMC3" %}
-{% set version = "3.1.0" %}
+{% set version = "3.1.1" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: http://hepmc.web.cern.ch/hepmc/releases/{{ name  }}-{{ version }}.tar.gz 
-  sha256: ac02977d6bde18398abf9d25a2e55614e22b5e819ed4b21ea55cf31aabe6df67
+  sha256: 2fcbc9964d6f9f7776289d65f9c73033f85c15bf5f0df00c429a6a1d8b8248bb
```

Comment:
```suggestion
  sha256: 46098a7e9717921e436c265836c914d515cb831dde898f3f1d129468df7982e7
```

Reply:
Oops, looks like I can't push to the bot's repository?

Reply:
Actually, it looks like the bot's suggestion is the right one... Now not sure why it is wrong in the CI...

Reply:
And not sure why GitHub won't let me commit a suggestion here, but it does let me push one. Oh well...